app/kudos/views.py

```diff
@@ -690,3 +691,45 @@ def receive_bulk(request, secret):
         'kudos_transfer': kudos_transfer,
     }
     return TemplateResponse(request, 'transaction/receive_bulk.html', params)
+
+
+@csrf_exempt
+def newkudos(request):
+    context = {
+        'active': 'newkudos',
+        'msg': None,
+    }
+
+    if request.POST:
+        required_fields = ['name', 'description', 'priceFinney', 'artist', 'platform', 'numClonesAllowed', 'tags', 'to_address', 'artwork_url']
+        validtion_passed = True
+        for key in required_fields:
+            if not request.POST.get(key):
+                context['msg'] = str(_('You must provide the following fields: ')) + key
+                validtion_passed = False
+        if validtion_passed:
+            # save / send email
+            obj = TokenRequest.objects.create(
+                name=request.POST['name'],
+                description=request.POST['description'],
+                priceFinney=request.POST['priceFinney'],
+                artist=request.POST['artist'],
+                platform=request.POST['platform'],
+                numClonesAllowed=request.POST['numClonesAllowed'],
+                tags=",".split(request.POST['tags']),
+                to_address=request.POST['to_address'],
+                artwork_url=request.POST['artwork_url'],
+                network='mainnet',
+                approved=False,
+                metadata={
+                    'ip': get_ip(request),
+                    'email': request.POST['email'],
```

Comment:
`email` isn't validated as a required field in https://github.com/gitcoinco/web/pull/2936/files#diff-9b7a42d8513c01971adb785752bd3f67R704 - We'll probably want to use `.get('key', 'default_value')` for all of the params that aren't required here.

Reply:
6865ed40568af4b3397f5a91b1c4d80870d95760